core/git_command.py

```diff
@@ -184,11 +184,16 @@ def git(self, *args,
 
             if show_panel and live_panel_output:
                 if savvy_settings.get("show_input_in_output"):
-                    util.log.panel(command_str + "\n\n")
+                    util.log.panel("> {}\n\n".format(command_str))
                 wrapper = LoggingProcessWrapper(p)
                 stdout, stderr = wrapper.communicate(stdin)
             else:
                 stdout, stderr = p.communicate(stdin)
+                if show_panel:
```

Comment:
stderr and stdout are no longer decoded before being shown in the panel. Intentional?

Reply:
Not,  Intentional.  
```pyhton
            if show_panel and live_panel_output:
                util.log.panel("")
                if savvy_settings.get("show_stdin_in_output") and stdin is not None:
                    util.log.panel_append("STDIN\n{}\n".format(original_stdin))
                if savvy_settings.get("show_input_in_output"):
                    util.log.panel_append("> {}\n\n".format(command_str))
                wrapper = LoggingProcessWrapper(p)
                stdout, stderr = wrapper.communicate(stdin)
            else:
                stdout, stderr = p.communicate(stdin)

            if decode:
                stdout, stderr = self.decode_stdout(stdout, savvy_settings), stderr.decode()

            if show_panel and !live_panel_output:
                if savvy_settings.get("show_input_in_output"):
                    util.log.panel("> {}\n{}\n{}".format(command_str, stdout, stderr))
                else:
                    util.log.panel("{}\n{}".format(stdout, stderr))
```